go/kbfs/libkbfs/prefetcher.go

```diff
@@ -873,6 +882,305 @@ func (p *blockPrefetcher) handleStatusRequest(req *prefetchStatusRequest) {
 	}
 }
 
+func (p *blockPrefetcher) handlePrefetchRequest(req *prefetchRequest) {
```

Comment:
This function is copy-pasted from the original `run()` function, except where noted.